source/use-cases/sensitive-data-encryption.txt

```diff
@@ -688,3 +725,21 @@ check out the reference docs in the server manual:
 - :manual:`Client-Side Field Level Encryption </core/security-client-side-encryption>`
 - :manual:`Automatic Encryption JSON Schema Syntax </reference/security-client-side-automatic-json-schema>`
 - :manual:`Manage Client-Side Encryption Data Keys </tutorial/manage-client-side-encryption-data-keys>`
+
+
+.. tabs-drivers::
+   :hidden: true
+
+   .. tab::
+      :tabid: java-sync
+      
+      See more information on `Client Side Encryption using the official
```

Comment:
Nitpick: I was confused by the order of words in this sentence.

I could incorrectly read this as "View _more information_ that is written about <this link>". I may be seeking more information, but might be non sequitur am not certain whether <this link> is the topic I am looking for.
Instead, I might phrase this with a different ordering of words:
"See <this link> for more information." which makes it clear that the link provided contains detail on the current discussion topic.
Hope that makes sense.

Reply:
I changed the phrasing to "For additional information on CSFLE, see the official Java driver documentation" per nick's suggestion above ^. Let me know what you think

Reply:
![image](https://user-images.githubusercontent.com/52428683/65919432-44385a00-e3aa-11e9-970e-0e10087f652e.png)
(sey-mour)